src/Tests/Seam/When_dispatching_messages_in_receive_context.cs

```diff
@@ -284,5 +216,34 @@ public class When_dispatching_messages_in_receive_context
             container.RegisterSingleton<TopologyOperator>();
             return sectionManager;
         }
+
+        void StartPumpWithConcurrencySetToOne()
+        {
+            // start the pump
+            pump.Start(new PushRuntimeSettings(1));
+        }
+
+        async Task StopPump()
```

Comment:
Personally I don't like helper methods like this that as they don't really save anything and do exactly the same. Unline `StartPumpWithConcurrencySetToOne(1)` where it is not intuitive that 1 stands for concurrency. Even then it could be written as  `pump.Start(new PushRuntimeSettings(concurrency: 1));` and helper method would not be required.